### ISSUE TITLE
NoSuchMethodError after upgrading to Play 2.7.9

### ISSUE THREAD

**Issue by jchernan** [kamon-io/Kamon#941]
Play 2.7.8 works fine, but changing to 2.7.9 fails with `java.lang.NoSuchMethodError`.

```
"io.kamon" %% "kamon-bundle" % "2.1.12",
"io.kamon" %% "kamon-apm-reporter" % "2.1.12"
```

```
addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.9")
addSbtPlugin("io.kamon" % "sbt-kanela-runner-play-2.7" % "2.0.7")
```

```
Uncaught error from thread [play-dev-mode-akka.actor.default-dispatcher-4]: 'better.files.Implicits$FileOps better.files.package$.FileOps(java.io.File)', shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled for ActorSystem[play-dev-mode]
java.lang.NoSuchMethodError: 'better.files.Implicits$FileOps better.files.package$.FileOps(java.io.File)'
	at kamon.instrumentation.sbt.play.KanelaReloader.$anonfun$reload$3(KanelaReloader.scala:474)
	at scala.collection.Iterator$$anon$11.nextCur(Iterator.scala:484)
	at scala.collection.Iterator$$anon$11.hasNext(Iterator.scala:490)
	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:458)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.TraversableOnce.foldLeft(TraversableOnce.scala:162)
	at scala.collection.TraversableOnce.foldLeft$(TraversableOnce.scala:160)
	at scala.collection.AbstractIterator.foldLeft(Iterator.scala:1429)
	at kamon.instrumentation.sbt.play.KanelaReloader.$anonfun$reload$1(KanelaReloader.scala:476)
	at kamon.instrumentation.sbt.play.KanelaReloader$$anon$1.run(KanelaReloader.scala:44)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at kamon.instrumentation.sbt.play.KanelaReloader$.kamon$instrumentation$sbt$play$KanelaReloader$$withReloaderContextClassLoader(KanelaReloader.scala:49)
	at kamon.instrumentation.sbt.play.KanelaReloader.reload(KanelaReloader.scala:462)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:142)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:188)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1262)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:523)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:409)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:606)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:485)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:581)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:749)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:764)
	at akka.actor.Actor.aroundReceive(Actor.scala:539)
	at akka.actor.Actor.aroundReceive$(Actor.scala:537)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:671)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:614)
	at akka.actor.ActorCell.invoke(ActorCell.scala:583)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:268)
	at akka.dispatch.Mailbox.run(Mailbox.scala:229)
	at kamon.instrumentation.executor.ExecutorInstrumentation$InstrumentedForkJoinPool$TimingRunnable.run(ExecutorInstrumentation.scala:678)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:49)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
[info] p.c.s.AkkaHttpServer - Stopping server...
```

**Comment by SimunKaracic** [comment]
Please bump the kanela-runner to 2.0.8 and try again!

**Comment by jchernan** [comment]
Thank you for the response! It doesn't seem like 2.0.8 has been published (I'm looking at https://bintray.com/kamon-io/sbt-plugins/sbt-kanela-runner).

```
[error] sbt.librarymanagement.ResolveException: Error downloading io.kamon:sbt-kanela-runner-play-2.7;sbtVersion=1.0;scalaVersion=2.12:2.0.8
[error]   Not found
```

**Comment by SimunKaracic** [comment]
Sorry, got mixed up. 

We ran into this issue with play 2.8 and fixed it in `2.0.7`.
`2.0.8` has just been released, fixing this for play 2.7 and 2.6!

**Comment by jchernan** [comment]
It is working now, thank you!
